disnake/ext/commands/slash_core.py

```diff
@@ -332,10 +332,12 @@ def parents(
 
     @property
     def description(self) -> str:
+        """:class:`str`: The slash sub command's description. Shorthand for :attr:`self.body.description <Option.description>`."""
```

Comment:
```suggestion
        """:class:`str`: The slash sub command's description. Shorthand for :attr:`self.body.description <.Option.description>`."""
```
sphinx is being nitpicky once more - seems like I accidentally changed this in one of the previous comments, my bad :c